### DISCUSSION TITLE
Detect users not sign-in with MFA, Misconfiguration for MFA

### DISCUSSION THREAD

**Discussion by sergioandreslq** [maester365/maester#443] [Ideas]
Is there a way in Maester to run and detect from all the members which one are not performing MFA?

I use MsIdentityTools, however, it tell me the users connecting to Azure Portal CLI or Powershell,  however, the objective is to know if we have users without MFA

**Comment by soulemike** [comment]
This is a good idea for a test especially with Authentication Methods.

https://learn.microsoft.com/en-us/graph/api/resources/authenticationmethods-usage-insights-overview

https://learn.microsoft.com/en-us/graph/api/authentication-list-methods

**Comment by robybrisson** [comment]
> Is there a way in Maester to run and detect from all the members which one are not performing MFA?
> 
> I use MsIdentityTools, however, it tell me the users connecting to Azure Portal CLI or Powershell, however, the objective is to know if we have users without MFA

I'd recommend a tool like Augmentt for that.

![image](https://github.com/user-attachments/assets/b0a19fb0-5d59-480e-9157-fe08725c5ea6)
